tests/test_integration.py

```diff
@@ -22,32 +25,34 @@
 from crate.crash.commands import Command
 from crate.crash.outputs import _val_len as val_len
 from crate.crash.printer import ColorPrinter
-from crate.testing.layer import CrateLayer
 from tests import ftouch
 
-if sys.platform != "linux":
-    raise SkipTest("Integration tests only supported on Linux")
-
-crate_version = os.getenv("CRATEDB_VERSION", "5.4.5")
-crate_http_port = 44209
-crate_transport_port = 44309
-crate_settings = {
-    'cluster.name': 'Testing44209',
-    'node.name': 'crate',
-    'psql.port': 45441,
-    'lang.js.enabled': True,
-    'http.port': crate_http_port,
-    'transport.tcp.port': crate_transport_port
-}
-node = CrateLayer.from_uri(
-    f'https://cdn.crate.io/downloads/releases/cratedb/x64_linux/crate-{crate_version}.tar.gz',
-    'crate',
-    settings=crate_settings
-)
+
+class EntrypointOpts:
+    version = os.getenv("CRATEDB_VERSION", "5.4.5")
+    psql_port = 45441
+    http_port = 44209
+    transport_port = 44309
+    settings = {
+        "cluster.name": "Testing44209",
+        "node.name": "crate",
+        "lang.js.enabled": True,
+        "psql.port": psql_port,
+        "http.port": http_port,
+        "transport.tcp.port": transport_port,
+    }
 
 
+node = CrateDBFixture(crate_version=EntrypointOpts.version)
```

Comment:
With respect to your question on the other patch, could this spot also be used to optionally register table names for resetting, on behalf of a second argument to the fixture manager here? In this way, it can be managed by `CrateDBFixture`, but the user has control about which tables need to be defined here.

The list of tables could be supplied to the constructor, or optionally by using a dedicated method, or both. It is just an idea, I don't have any strong opinions about the "how".
```python
node.register_tables_for_reset(*tablenames: t.List[str])
```

Reply:
You already responded to this at https://github.com/crate-workbench/cratedb-toolkit/pull/82#discussion_r1402805521.

Your proposed interface is perfectly lean.
```python
cratedb_service.reset(tables=RESET_TABLES)
```

Reply:
Yes, I think passing the table list to the `reset` method is even better than using it in the class constructor/class method, as you can adjust the list with more flexibility with no need to change the main fixture.